settings/authentication-personalization/authentication-vs-personalization.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Authentication vs Personalization'
+description: 'How to determine which product is best for you'
```

Comment:
```suggestion
description: 'How to determine which feature is best for you'
```

Reply:
I decided to call it a product because it's in the "products" page in the dashboard and it's easier to talk about the different features of Personalization if Personalization is a "product" and all the things you can do with Personalization are "features"

Reply:
gotchu